docs/modules/ROOT/pages/_partials/calling-fuse-aggregation-app-endpoint-fail-limits.adoc

```diff
@@ -4,22 +4,27 @@
 
 
 [id='calling-fuse-aggregation-app-endpoint-fail-limits_{context}']
-= calling-fuse-aggregation-app-endpoint-fail-limits 
+= Verifying access to the API Service is limited 
 
-. In the *user_key* field, enter `{fuse-aggregator-app-name}`
+. In the *user_key* field, enter `{fuse-aggregator-app-name}`.
 . Click the *Try it out!* button repeatedly until the *Response Code* is *0*, this
-should require less than 5 clicks (the hourly limit set earlier)
-. Select the *Applications* tab from the top menu
-. Select the *{fuse-aggregator-app-name}* application from the *Applications* list
-. Scroll down to the *Current Utilization* section
+should require less than 5 clicks (the hourly limit set earlier).
+. Select the *Applications* tab from the top menu.
+. Select the *{fuse-aggregator-app-name}* application from the *Applications* list.
+. Scroll down to the *Current Utilization* section.
 
 ifdef::location[]
 
-.Verification
+.To verify this procedure:
```

Comment:
@finp Just noticed that only the content in the `tag::verification[]` block is shown in the webapp.
The heading is not shown.
I can create a bug/issue for this if you're in agreement it should be shown (I don't see why not).

![image](https://user-images.githubusercontent.com/878251/47149065-6f758680-d2ca-11e8-87d0-d4074f912b15.png)

It could be formatted like the way 'Check your work' is currently formatted (the webapp could do that).